fix_laser_sampling _errors.py

```diff
@@ -0,0 +1,75 @@
+#How to change value in hf5 File due to Laser sampling error (high sampling rate)
+
+import tables
+import numpy as np
+import easygui
+import os
+
+# Ask for the directory where the hdf5 file sits, and change to that directory
+dir_name = easygui.diropenbox()
+os.chdir(dir_name)
+
+# Get the names of all files in the current directory, and find the hdf5 (.h5) file
+file_list = os.listdir('./')
+hdf5_name = ''
+for files in file_list:
+	if files[-2:] == 'h5':
+		hdf5_name = files
+
+# Open up the hdf5 file
+hf5 = tables.open_file(hdf5_name, 'r+')
+
+trains_dig_in = hf5.list_nodes('/spike_trains')
+
+# Ask the user for the correct laser duration and convert to integers
+num_durations = easygui.multenterbox(msg = 'how many laser durations were used in this experiment?', fields = ['Number of Laser durations'])
+num_durations = int(num_durations[0])
+
+durations = easygui.multenterbox(msg = 'What are the laser durations used in this experiment?', fields = ["Duration" for num in range(num_durations)])
+for i in range(len(durations)):
+    durations[i] = int(durations[i])
+    durations.append(0)

```

Comment:
The for loop should look like this:
```
for i in range(len(durations)):
   durations[i] = int(durations[i])
durations.append(0)
```
In your version, you will be appending 0 to the durations list everytime the for loop runs. This works right now because you only have 1 duration - but if you have more durations, then multiple zeros will get appended.